app/src/main/java/ch/epfl/balelecbud/cloudMessaging/Message.java

```diff
@@ -0,0 +1,100 @@
+package ch.epfl.balelecbud.cloudMessaging;
+
+import android.util.Log;
+
+import androidx.annotation.VisibleForTesting;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import ch.epfl.balelecbud.BalelecbudApplication;
+import ch.epfl.balelecbud.models.User;
+import ch.epfl.balelecbud.util.database.DatabaseWrapper;
+import ch.epfl.balelecbud.util.database.FirestoreDatabaseWrapper;
+
+import static ch.epfl.balelecbud.BalelecbudApplication.getAppAuthenticator;
+import static ch.epfl.balelecbud.BalelecbudApplication.getAppDatabaseWrapper;
+
+public class Message {
+
+    private static final String TAG = CloudMessagingService.class.getSimpleName();
```

Comment:
```suggestion
    private static final String TAG = Message.class.getSimpleName();
```

Reply:
The class Message only exists for the CloudMessagingService, they have the same tag so in it is easier to find the CloudMessaging errors/messages in the Logcat

Reply:
I side with @Pialex99 on this one